src/modules/nav/nav.component.html

```diff
@@ -2,6 +2,11 @@
   <ol class="stache-nav-list">
     <li class="stache-nav-list-item" *ngFor="let route of routes">
       <a class="stache-nav-anchor" (click)="navigate(route)" [ngClass]="{'active': isActive(route)}">{{route.name}}</a>
+      <ol class="stache-nav-child-list" *ngIf="hasChildRoutes(route)">
```

Comment:
I like the `hasChildRoutes` method. Possible to inject another `<stache-nav></stache-nav>` in here, instead of generating a new `ol`?